scripts/generate_emg_test_data.py

```
@@ -0,0 +1,307 @@
+    # Parameters - peak center BEYOND truncation
+    height = 50000.0
+    width = 5.0
+    symmetry = 5.0
+    true_retention = 210.0  # Peak center BEYOND truncation at 200
+
+    # Data truncated at 200 - we only see the rising edge of the peak
+    t = [100.0 + i for i in range(101)]  # 100 to 200
+    y_data = emg_function(t, height, width, symmetry, true_retention)
+
+    print(f"True RT = {true_retention} (BEYOND truncation at 200)")
+    print(f"Data range: [100, 200]")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)